jobbergate-cli/jobbergate_cli/render.py

```diff
@@ -188,7 +192,10 @@ def render_single_result(
     :param: title:         The title header to include above the ``Tale`` output
     """
     if isinstance(result, pydantic.BaseModel):
-        result = result.dict()
+        result_model = cast(pydantic.BaseModel, result)
+        result_dict = json.loads(result_model.json())
+        result = cast(Dict[str, Any], result_dict)
+
```

Comment:
why??

Reply:
The following snippet shows it is supposed to be the same thing but this implementation is not that good

```python
from pydantic import BaseModel
import json
from typing import Any, Dict, cast
import time

class Foo(BaseModel):
    foo: str

foo = Foo(foo="boo")

init = time.time()
foo_dict_easy = foo.dict()
finish = time.time()
print("Time for easy operation: {} s".format(finish - init))


init = time.time()
result_model = cast(BaseModel, foo)
result_dict = json.loads(result_model.json())
result = cast(Dict[str, Any], result_dict)
finish = time.time()
print("Time for complex operation: {} s".format(finish - init))
print()

print("Are both results equal? {}".format(foo.dict() == result))
```